web3/contracts.go

```diff
@@ -145,9 +162,42 @@ func (c *Contracts) LoadContracts(addresses *Addresses) error {
 	if err != nil {
 		return fmt.Errorf("failed to bind process registry: %w", err)
 	}
+
+	ctx, cancel := context.WithTimeout(context.Background(), web3QueryTimeout)
+	defer cancel()
+
+	// check vkey used by sequencer is the same as the one used by the contract
```

Comment:
No, we are not checking this, but instead we are checking that the PROVING key on the sequencer is compatible with the Verification key on the smart contract.